docs/data/rpc/api-reference/methods/README.mdx

```diff
@@ -8,3 +8,20 @@ import DocCardList from "@theme/DocCardList";
 All you need to know about available RPC methods, parameters and responses, and making RPC requests.
 
 <DocCardList />
+
+Don't know which endpoint you need to get what you want? A lot of the returned fields are deeply-nested [XDR structures](/docs/learn/encyclopedia/data-format/xdr.mdx#more-about-xdr), so it can be hard to figure out what kind of information is available in each of these. Here's a bit of a dive into what the "workhorse" endpoints provide, in decreasing order of granularity:
+
+- [`getLedgers`](./getLedgers.mdx) operates at the block level, providing you with the full, complete details of what occurred during application of that ledger (known as "ledger metadata", defined in the protocol by the [`LedgerCloseMetaV1`](https://github.com/stellar/stellar-xdr/blob/v22.0/Stellar-ledger.x#L508-L537) structure). Each of the subsequent endpoints is just a microscope into a subset of the data available provided by this endpoint. Metadata includes things like:
```

Comment:
I think we should link to LedgerCloseMeta instead of LedgerCloseMetaV1 because technically that is the XDR we return in the `getLedgers()` endpoint. From the current description, I think it would be easy to make the mistake that you should parse the `getLedgers()` output into a `LedgerCloseMetaV1` instance instead of a `LedgerCloseMeta` instance

Reply:
Yeah I was conflicted on this. I linked to V1 so that a reader could see specifically what kinds of fields were available immediately rather than needing to understand the whole V0/V1 thing and mitigated the actual decoding confusion by linking to the parent union on the [endpoint page](https://developers-pr1369.previews.kube001.services.stellar-ops.com/docs/data/rpc/api-reference/methods/getLedgers) itself.

You're right that this inconsistency isn't ideal, though. I relinked+reworded it in 39afe62890ac4d824990a0ce9bb484d8422f4828!